importers/src/main/groovy/whelk/importer/WhelkCopier.groovy

```diff
@@ -67,18 +89,17 @@ class WhelkCopier {
         def newId = dest.baseUri.resolve(doc.shortId).toString()
         newDoc.id = newId
 
-        if (dest.storage.getMainId(newId)) {
-            dest.storeAtomicUpdate(newDoc.getShortId(), true, "xl", "WhelkCopier") {
-                it.data = newDoc.data
+        def collection = LegacyIntegrationTools.determineLegacyCollection(newDoc, dest.jsonld)
+        if (collection && collection != "definitions") {
```

Comment:
Will LegacyIntegrationTools.determineLegacyCollection() ever return collection = "definitions"? When I was debugging this previously, it returned null for definitions-records.

Reply:
You're probably right, but this doesn't hurt and (I think that) it more clearly communicates the intent to the reader this way.